index.html

```diff
@@ -14,30 +28,26 @@
 				<li class="nav-item icon-user-tie" data-content="about"> <a href="#"> About </a> </li>
 			</ul>
 		</div>
-		<div id="middle">
-			<div id="home" class="swapped-body-content">
-				<h1> Welcome</h1>
-				<p> Use the navigation bar to checkout my projects and learn a little about me. </p>
-			</div>
-			<div id="about" class="swapped-body-content">
-				<div id="profile-picture-holder">
-					<img id="profile-picture" src="Assets/profilePicture.jpg">
+		<div id="right-side">
+			<div id="middle">
+				<div id="home" class="swapped-body-content">
+					<h1> Welcome</h1>
+					<p> Use the navigation bar to checkout my projects and learn a little about me. </p>
 				</div>
-				<p id="intro"> Hi, my name is Chad Oehlschlaeger-Browne. </p>
-				<a class="social" href="https://github.com/chadob"> My GitHub Portfolio </a>
-				<a class="social" href="https://www.linkedin.com/nhome/?trk="> My Linked-In </a>
-			</div>
-			<div id="projects" class="swapped-body-content">
-				<div class ="template project">
-					<a class="picture-link" href="">
-						<img class="project-picture" src="">
-					</a>
-					<p class="project-par"></p>
-					<p> <time class="project-date"></time> </p>
+				<div id="about" class="swapped-body-content">
+					<div id="profile-picture-holder">
+						<img id="profile-picture" src="Assets/profilePicture.jpg">
```

Comment:
Make sure to add "alt" attributes to images for people with vision impairment and web crawlers.